src/script/api/script_info_docs.hpp

```diff
@@ -218,17 +218,23 @@ class ScriptInfo {
 	 *    store the current configuration of Scripts. Required.
 	 *  - description A single line describing the setting. Required.
 	 *  - min_value The minimum value of this setting. Required for integer
-	 *    settings and not allowed for boolean settings.
+	 *    settings and not allowed for boolean settings. The value will be
+	 *    clamped in the range [-2147483648, 2147483647] (inclusive).
 	 *  - max_value The maximum value of this setting. Required for integer
-	 *    settings and not allowed for boolean settings.
+	 *    settings and not allowed for boolean settings. The value will be
+	 *    clamped in the range [-2147483648, 2147483647] (inclusive).
 	 *  - easy_value The default value if the easy difficulty level
-	 *    is selected. Required.
+	 *    is selected. Required. The value will be clamped in the range
+	 *    [-2147483648, 2147483647] (inclusive).
 	 *  - medium_value The default value if the medium difficulty level
-	 *    is selected. Required.
+	 *    is selected. Required. The value will be clamped in the range
+	 *    [-2147483648, 2147483647] (inclusive).
 	 *  - hard_value The default value if the hard difficulty level
-	 *    is selected. Required.
+	 *    is selected. Required. The value will be clamped in the range
+	 *    [-2147483648, 2147483647] (inclusive).
 	 *  - custom_value The default value if the custom difficulty level
-	 *    is selected. Required.
+	 *    is selected. Required. The value will be clamped in the range
+	 *    [-2147483648, 2147483647] (inclusive).
 	 *  - random_deviation If this property has a nonzero value, then the
 	 *    actual value of the setting in game will be randomized in the range
 	 *    [user_configured_value - random_deviation, user_configured_value + random_deviation] (inclusive).
```

Comment:
This one is also clamped but to `0 .. MAX(int32)`

Reply:
It's not exactly "clamped". I just don't know how to describe it. It's Clamp(abs(res)) in code. -30 becomes 30, not 0.

Reply:
Sign is ignored, and it's clamped.